### ISSUE TITLE
Hyperspace not 100x

### ISSUE THREAD

**Issue by dconorozzo** [LemADEC/WarpDrive#360]
Hyperspace is only moving 4x for me.  Am I missing something? I am using warpdrive version 1.4.2 in Minecraft 1.7.10. Is there a config somewhere I can change?  Thanks!

**Comment by LemADEC** [comment]
Where did you read a 100x factor?

**Comment by dconorozzo** [comment]
https://github.com/LemADEC/WarpDrive/wiki
Hyperspace movement is typically 10x more expensive while being 100x faster. Each transition to or from hyperspace requires 10M EU.

https://minecraft.curseforge.com/projects/warpdrive
- New hyperspace dimension allowing to move 100x faster then in space. Your ship needs to be big enough to reach hyperspace, or you'll need to use jump-gates to move in/out of hyperspace.

Sorry.  I gather from your reply that I am misinterpreting that...  Is there a config adjustment somewhere? I imported the galacticraft and more planets mod planets in and set them up in the warpdrive celestial bodies config but the distances are sometimes huge (I started out trying to preserve real life distances to some degree).
